src/famfs_lib.c

```diff
@@ -3786,16 +3852,19 @@ int
 __famfs_mkfs(const char              *daxdev,
 	     struct famfs_superblock *sb,
 	     struct famfs_log        *logp,
+	     u64                      log_len,
 	     u64                      device_size,
 	     int                      force,
 	     int                      kill)
 
 {
 	int rc;
 
-	if ((famfs_check_super(sb) == 0) && !force) {
-		fprintf(stderr, "Device %s already has a famfs superblock\n", daxdev);
-		return -1;
+	/* Minimum log length is the FAMFS_LOG_LEN; Also, must be a power of 2 */
+	if (log_len & (log_len - 1) || log_len < FAMFS_LOG_LEN) {
+		fprintf(stderr, "%s: log length (%lld) not a 2MiB multiple\n",
```

Comment:
Since the check expects that it is to be a power of 2 value, the error message can be 
"log length (%lld)  should be greater than 2MiB and a power of 2 value"
Multiple of 2MiB is also 10MiB which is not power of 2...